src/librustdoc/json/conversions.rs

```diff
@@ -546,13 +545,13 @@ impl FromWithTcx<clean::Import> for Import {
             Simple(s) => Import {
                 source: import.source.path.whole_name(),
                 name: s.to_string(),
-                id: import.source.did.map(FakeDefId::from).map(from_def_id),
+                id: import.source.did.map(Into::into).map(from_def_id),
```

Comment:
Why this change?

Reply:
I first removed the `.map(FakeDefId::from)` part and needed to fix it later, which I used `Into::into`, but I agree that `FakeDefId::from` is better.